pkg/convert/angular_localize.go

```diff
@@ -0,0 +1,124 @@
+package convert
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+// extract-i18n guide and specification: https://angular.io/guide/i18n-common-translation-files
+// extract-i18n JSON example: https://github.com/angular/angular/issues/45465
+// extract-i18n XLF example: https://angular.io/guide/i18n-common-translation-files#translate-each-translation-file
+
+// ----------------------JSON--------------------------
+
+// Extracted messages with json format contains only id:message.
+type ngJSON struct {
+	Language     language.Tag      `json:"locale"`
+	Translations map[string]string `json:"translations"`
+}
+
+func (n ngJSON) fromNgJSON() model.Messages {
+	messages := model.Messages{
+		Language: n.Language,
+		Messages: make([]model.Message, 0, len(n.Translations)),
+	}
+
+	for k, v := range n.Translations {
+		msg := model.Message{ID: k, Message: v}
+		messages.Messages = append(messages.Messages, msg)
```

Comment:
Personally, I don't bother declaring a new variable unless it adds some clarity and improves readability. Up to you what to do.

```suggestion
		messages.Messages = append(messages.Messages, model.Message{ID: k, Message: v})
```